apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -835,6 +1380,42 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
                     </Grid.Row>
                 </Grid>
             </Modal.Actions>
+        );
+    };
+
+    return (
+        <Modal
+            data-testid={ testId }
```

Comment:
Better to use `componentid` on newer components

Reply:
This will be fixed after the release by revamping all the components to use componentid.